helm/values-dev.yaml

```diff
@@ -86,6 +86,7 @@ microservice-chart:
     WEBSITE_SITE_NAME: 'pagopa-d-gpd-core-service'
     ENV: 'dev'
     SPRING_DATASOURCE_URL: "jdbc:postgresql://gpd-db.d.internal.postgresql.pagopa.it:6432/apd?sslmode=require&prepareThreshold=0&tcpKeepAlive=true"
+    FLYWAY_DATASOURCE_URL: "jdbc:postgresql://gpd-db.d.internal.postgresql.pagopa.it:6432/apd?sslmode=require&prepareThreshold=0&tcpKeepAlive=true&lock_timeout=30000"
```

Comment:
why add `FLYWAY_DATASOURCE_URL` if isnt'used ?`
and for flyway migration doesn't use `6432` ( pgbouncer ) but directly `5432`